website/hugo.yaml

```diff
@@ -28,6 +28,10 @@ permalinks:
   # TODO(bwplotka): Move to mdox + fixes to ensure we don't need to manually do this.
   # For now we added few entries for advance.
   tip: "/:sections/:filename.md"
+  v0.33: "/:sections/:filename.md"
```

Comment:
I wonder if it is fine to have v0.33 since it is not released yet

Reply:
Should I remove this line then?